CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,287 @@
+# v1.10.0 - Changelog since v1.9.2
+
+## Changes by Kind
+
+### Feature
+- It is no longer necessary to specify allowedTopologies for zonal -> zonal cloning and zonal/regional -> regional cloning to ensure that the clone zone is compatible with the GCE volume cloning requirements. ([#1150](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1150), [@amacaskill](https://github.com/amacaskill))
+
+### Bug or Regression
+
+- Fix missing libedit.so.2 error ([#1177](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1177), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Fixes CVE-2022-41723 ([#1169](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1169), [@artemvmin](https://github.com/artemvmin))
```

Comment:
Hm the release note doesn't really match the main intent of the change, which is to resolve the mkfs OOM issue. Maybe this change needs two release notes.

Reply:
Ok will update

Reply:
I broke it into 3 based on his commits, but I don't really understand what the "break dependency on k/k commit" is I'll ask him.